### PR TITLE
Update sector list to reflect recent revisions

### PR DESCRIPTION
**Pull request by steventux**
https://trello.com/c/WoCAgoHq/153-improve-the-sector-list

The 'sector business area' facet values have been revised to give better
filtering options in the business readiness finder. This ports these changes
to the new facet content item structure.

### CODE REVIEW

File: lib/data/find-eu-exit-guidance-business.yml [hunk 1]
```diff
@@ -13,134 +13,188 @@
...
         :title: Automotive
         :value: automotive
-      - :content_id: 3880f354-3d74-4441-b40f-d808933a14be
-        :title: Banking, markets and infrastructure
-        :value: banking-market-infrastructure
-      - :content_id: 4c7a04d8-02b8-430c-bfec-f1e1f475ff98
-        :title: Broadcasting
-        :value: broadcasting
+      - :content_id: 5faa1741-fc55-4110-b342-de92f6324118
+        :title: Auxillary services
```
Comment (vanitabarrett) [inline lib/data/find-eu-exit-guidance-business.yml]: Typo: should be `Auxiliary`. Same for the slug too

File: lib/data/find-eu-exit-guidance-business.yml [hunk 2]
```diff
@@ -13,134 +13,188 @@
...
+      - :content_id: cbe95935-13c7-4de8-8d3c-b7c9ef8d69a2
+        :title: Non-metal materials manufacture
+        :value: non-metal-materials-manufacture
+      - :content_id: 45bfb0a2-b60d-40f2-b9af-b5d2b4b91445
+        :title: Nuclear
+        :value: nuclear
       - :content_id: 46ad5a1d-fc2d-4065-a34f-3f7b2fea8e81
-        :title: Oil and gas production
-        :value: oil-and-gas-production
+        :title: Oil gas and coal
```
Comment (vanitabarrett) [inline lib/data/find-eu-exit-guidance-business.yml]: Needs a comma: 'Oil, gas and coal'

**Comment by vanitabarrett** [comment]
Is there a reason why the content id for Clothing and consumer goods is shorter than the others?

```
:content_id: ec4-1a6e-431f-9199-4207bf8e9580
:title: Clothing and consumer goods
```

File: lib/data/find-eu-exit-guidance-business.yml [hunk 3]
```diff
@@ -13,134 +13,194 @@
...
         :value: retail
-      - :content_id: a9736af7-4e8b-4b1f-9a4b-9482252f4a4f
-        :title: Social work
-        :value: social-work
-      - :content_id: 99eeb13a-ce9d-49f6-aa5b-68d0e4a45358
-        :title: Steel and other metals or commodities
-        :value: steel-and-other-metals-commodities
+      - :content_id: 6f5f8dbb-3cd3-475d-8f3a-ed6e22d370be
+        :title: Road (passengers and freight)
+        :value: road-passenger-freight
```
Comment (vanitabarrett) [inline lib/data/find-eu-exit-guidance-business.yml]: This should be `road-passengers-freight`

Comment (vanitabarrett) [inline lib/data/find-eu-exit-guidance-business.yml]: My mistake, this should be `road-passengers-freight`. I'll update my PRs to reflect that too.

File: lib/data/find-eu-exit-guidance-business.yml [hunk 4]
```diff
@@ -13,134 +13,194 @@
...
-        :title: Warehousing and support for transportation
-        :value: warehousing-and-support-for-transportation
-      - :content_id: 66c14974-d67e-4361-a14a-01804351ebd7
-        :title: Water transport including maritime and ports
-        :value: water-transport-maritime-ports
-      - :content_id: a788c432-0daa-4514-ab5b-777d013354d6
-        :title: Wholesale (excluding motor vehicles)
-        :value: wholesale-excl-motor-vehicles
+        :title: Warehousing, services and pipelines
+        :value: warehousing-services-pipelines
```
Comment (vanitabarrett) [inline lib/data/find-eu-exit-guidance-business.yml]: Should be `warehouses-services-pipelines`

**Comment by steventux** [comment]
@vanitabarrett I think this is up to date now.

**Pull request merged**